src/Native/Unix/System.Security.Cryptography.Native/pal_bio.cpp

```diff
@@ -52,3 +118,29 @@ extern "C" int32_t CryptoNative_BioCtrlPending(BIO* bio)
     assert(result <= INT32_MAX);
     return static_cast<int32_t>(result);
 }
+
+extern "C" void CryptoNative_BioSetAppData(BIO* bio, void* data)
+{
+    BIO_set_ex_data(bio, 0, data);
+}
+
+extern "C" void CryptoNative_BioSetWriteFlag(BIO* bio)
+{
+    BIO_set_flags(bio, BIO_FLAGS_WRITE);
+}
+
+extern "C" void CryptoNative_BioSetShoudRetryReadFlag(BIO* bio)
+{
+    BIO_set_flags(bio, BIO_FLAGS_SHOULD_RETRY | BIO_FLAGS_READ);
+}
```

Comment:
> I'll oppose Steve's request.

Wasn't a request, rather a question. :smile:  But it'd be a shame to find ourselves with lots of these methods that all differ only by what flags they pass.  If we believe we'll only ever have these two, fine.